manifest.yml

```diff
@@ -17,4 +17,4 @@ applications:
     services:
     - voice-of-interconnect-cloudant
     - voice-of-interconnect-speech-to-text
-    - voice-of-interconnect-natural-language-understanding
```

Comment:
oh that looks like something wrong on our side, I’d prefer to call it `natural-language-understanding` everywhere to avoid confusion in future. Can you explain why it had to change?